job-transifex-push.yml

```diff
@@ -0,0 +1,41 @@
+# Pushes string files to Transifex.
+
+parameters:
+- name: source_file_pattern
+  type: string
+  default: ".*/res/values/strings"
+- name: tx_token
+  type: string
+- name: transifex_client_version
+  type: string
+  default: "0.13.11"
+- name: twist_install_id
+  type: string
+- name: twist_install_token
+  type: string
+
+jobs:
+  - job: TransifexPush
+    pool:
+      vmImage: 'macOS-10.15'
+    timeoutInMinutes: 20
+    steps:
+    - checkout: self
+      submodules: false
+      persistCredentials: true
+    - script: |
+        if [ `git log -m -1 --name-only --pretty="format:" | grep "${{ parameters.source_file_pattern }}" | wc -l` -gt 0 ]; then
+          pip install transifex-client==${{ parameters.transifex_client_version }}
+          tx push -s
+        fi;
+      env:
+        TX_TOKEN: ${{ parameters.tx_token }}
+      failOnStderr: true
+      displayName: 'Push strings to Transifex'
+    - script: |
```

Comment:
This step is very useful for us, but it seems unexpected in a general-purpose "push to Transifex" job. 🤔 

Maybe we could split it into two yaml files with steps? I.e. `step-transifex-push.yml` (without posting to Twist) and `step-report-twist.yml` would be a more generic post-to-Twist step that also has a parameter for the content and we could then also reuse it for the build status reporting step.

Reply:
Yes, good idea 👍

If that's ok, I'll do that during the next iteration when adding a step to ping Dani.

Reply:
Sure!